pkgs/misc/drivers/hplip/default.nix

```diff
@@ -144,10 +146,30 @@ pythonPackages.buildPythonApplication {
     rm $out/etc/udev/rules.d/56-hpmud.rules
   '';
 
-  postFixup = ''
-    wrapProgram $out/lib/cups/filter/hpps \
-      --prefix PATH : "${nettools}/bin"
+  # The installed executables are just symlinks into $out/share/hplip,
+  # but wrapPythonPrograms ignores symlinks. We cannot replace the Python
```

Comment:
@abbradar you recall whether that is on purpose?

Reply:
I can answer that. `wrapPythonPrograms` ignores symlinks because you have to do this gymnastics of patching the target, deleting the symlink, and creating a wrapper directly to the target. I think it doesn't want to implement this logic because it assumes you can just call `wrapPythonPrograms` on the actual target, but we can't do that here.